java/integTest/src/test/java/glide/cluster/ClusterTransactionTests.java

```diff
@@ -103,6 +103,7 @@ public void keyless_transactions_with_group_of_commands(
 
     @SneakyThrows
     @Test
+    @Timeout(30)
```

Comment:
This needs to be reverted. It is slow now because we are using the ArgsArray protobuf field instead of the ArgsValuePointer. Same for TransactionTests.

Reply:
See my previous comment. I am going to comment theses tests until we complete the functionality phase.